clients/privacy-center/packages/fides-consent/src/fides-consent.ts

```diff
@@ -25,9 +26,19 @@ const defaults = makeDefaults({
  */
 const consent = getConsentCookie(defaults);
 
+/**
+ * Create a wrapped `banner` function that injects the default consent settings
+ * as an argument, so it can be used as Fides.banner()
+ */
+const banner = async (
+  extraOptions?: ConsentBannerOptions
+): Promise<void> => initBanner.call(undefined, defaults, extraOptions);
+
```

Comment:
This is really only done because `./lib/cookie.ts` is currently designed to not "know" what the configured defaults are, so we need to drill this value down through the banner when it goes to save an updated cookie. 

Basically, when "Accept All" is selected, we want to invoke `setConsentCookie` with a valid config object, but to build that config we need to provide some default values that are only known here, in `fides-consent.ts`. So I just drill them down into `Fides.banner()` but hide that using some `.call()` semantics...

Open to better ideas about how to manage this global state...

Reply:
Note to self: see convo here about the same thing... https://github.com/ethyca/fides/pull/2705#discussion_r1119222994